src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerExecDecorator.java

```diff
@@ -72,7 +70,6 @@ public ContainerExecDecorator(KubernetesClient client, String podName, String co
         this.client = client;
         this.podName = podName;
         this.containerName = containerName;
-        this.path = path;
```

Comment:
@carlossg there is only one usage of this constructor, so I can just change the signature inplace.
Do I still need to deprecate and create new one?